examples/miscellaneous/plot_set_output.py

```diff
@@ -109,3 +109,8 @@
 log_reg = clf[-1]
 coef = pd.Series(log_reg.coef_.ravel(), index=log_reg.feature_names_in_)
 _ = coef.sort_values().plot.barh()
+
+# %%
+# This resets `transform_output` to its default value to avoid impacting other
+# examples when generating the scikit-learn documentation
+set_config(transform_output="default")
```

Comment:
This is surprising to me. I thought the examples run in separate processes.

Reply:
As far as I know this is not the case: there is an `exec` somewhere that runs all the code from all the examples in the same interpreter. For example this is why sphinx-gallery has some reset functionality to clean-up before each example (e.g. closing all matplotlib figures): https://sphinx-gallery.github.io/stable/advanced.html#resetting-before-each-example